iceoryx_hoofs/README.md

```diff
@@ -155,28 +167,28 @@ setTimeout(11_s); // 11 seconds
 setTimeout(5_ms); // 5 milliseconds
 ```
 
-| class                   | internal | maybe obsolete | description |
-|:-----------------------:|:--------:|:--------------:|:------------|
-|`Duration`           | i | | Represents the unit time, is convertable to `timespec`, `timeval` and `std::chrono`. Uses user defined literals for readability.  |
+| class               | internal | maybe obsolete | description |
+|:-------------------:|:--------:|:--------------:|:------------|
+|`Duration`           | i |   | Represents the unit time, is convertible to `timespec` and `timeval`. User defined literals are available for convenience and readability.  |
 
 ### objectpool
 
-| class                   | internal | maybe obsolete | description |
-|:-----------------------:|:--------:|:--------------:|:------------|
-|`ObjectPool`           | i | | Container which stores raw objects without calling the ctor of the objects.  |
+| class                 | internal | maybe obsolete | description |
+|:---------------------:|:--------:|:--------------:|:------------|
+|`ObjectPool`           | i |   | Container which stores raw objects without calling the ctor of the objects.  |
 
 ### graphs
 
 | class                   | internal | maybe obsolete | description |
 |:-----------------------:|:--------:|:--------------:|:------------|
-|`DirectedGraph`          | i | | Creates and manages a [directed graph](https://en.wikipedia.org/wiki/Directed_graph).  |
-|`DirectedAcyclicGraph`   | i | | Like the DirectedGraph but additional checks prohibit to create edges which have a cyclic dependency.  |
+|`DirectedGraph`          | i |   | Creates and manages a [directed graph](https://en.wikipedia.org/wiki/Directed_graph).  |
```

Comment:
Where do we use this? Would it make sense to make this obsolete?

Reply:
So far I had in mind to wrap all "maybe obsolete" building blocks their own namespace, to be able to detect via `objdump` if some of those classes have been added to the final binary via `tools/scripts/analyse_blacklisted_namespaces_in_binary.sh`. Any objections to this idea?